CONTRIBUTING.md

```diff
@@ -112,3 +112,7 @@ The core team regularly looks at pull requests. We will provide
 feedback as soon as possible. After receiving our feedback, please respond
 within two weeks. After that time, we may close your PR if it isn't
 showing any activity.
+
+## Support
+
+Maintainers can provide limited support to contributors on discord: https://discord.gg/rWtp5Aj
```

Comment:
THis seems to be taking me to a server called "Grumpy gopher"?

Reply:
You are absolutely correct, thank you for catching this. I messed up the copy-paste and used the old discord link. It should be updated right now.